### PR TITLE
App demonstrating Serverless Testing

### PR DESCRIPTION
**Pull request by raghu12133**
## Serverless Testing

A sample app to demonstrate serverless testing through the test framework.

This app demonstrates the following features

1. Test suite structure.
2. Usage of test interfaces (invoke & stub).
3. Usage of `devDependencies` to include additional testing tools.

### CODE REVIEW

File: Play-Along/Freshhuddle/April-2020/serverless-testing/server/server.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    const subject = args.data.ticket.subject;
+
+    return $request.post({
+      url: 'https://httpbin.org/post',
+      json: {
+        ticketId,
+        subject
+      }
+    }).then(function() {
+      console.log('Request successful');
```
Comment (ravirajthedeveloper) [inline Play-Along/Freshhuddle/April-2020/serverless-testing/server/server.js:27]: Could you change this into `console.info`?

File: Play-Along/Freshhuddle/April-2020/serverless-testing/server/server.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      helper.setStatus(ticketId, true);
+    }, function() {
+      console.log('Request failed');
```
Comment (ravirajthedeveloper) [inline Play-Along/Freshhuddle/April-2020/serverless-testing/server/server.js:30]: Could you change this into `console.error` statement?
Also, the error object can be printed for debugging purpose when something goes wrong with the request.

File: Play-Along/Freshhuddle/April-2020/serverless-testing/server/test_data/onTicketCreate.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      "language": "en",
+      "created_at": "2016-12-12T09:37:47Z"
+    }
+  },
+  "timestamp": 1496400354326,
+  "account_id": 13,
+  "domain": "sample.freshdesk.com",
+  "event": "onTicketCreate",
+  "region": "US"
+}
```
Comment (ravirajthedeveloper) [inline Play-Along/Freshhuddle/April-2020/serverless-testing/server/test_data/onTicketCreate.json:75]: Add a new line at EOF here.

File: Play-Along/Freshhuddle/April-2020/serverless-testing/config/iparam_test_data.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ravirajthedeveloper) [inline Play-Along/Freshhuddle/April-2020/serverless-testing/config/iparam_test_data.json:1]: This file can be removed as it is unused and it is deprecated as well.

File: Play-Along/Freshhuddle/April-2020/serverless-testing/server/test_data/onAppInstall.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (ravirajthedeveloper) [inline Play-Along/Freshhuddle/April-2020/serverless-testing/server/test_data/onAppInstall.json:1]: This file can be removed as this event is unused.

**Comment by Saif-Shines** [comment]
All the code is almost fine. Can we have README.md file updated to the [standard format as mentioned here](https://github.com/freshdesk/marketplace-sample-apps/blob/master/.github/PULL_REQUEST_TEMPLATE/APP_README_TEMPLATE.md)?

**Pull request merged**